src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -3821,6 +3821,64 @@ internal new NamedTypeSymbol CreateNativeIntegerTypeSymbol(bool signed)
             return this.AnonymousTypeManager.ConstructAnonymousTypeSymbol(descriptor).GetPublicSymbol();
         }
 
+        protected override bool IsLanguageSpecificSupportedBuiltinOperatorType(ITypeSymbol type)
+        {
+            // ReadOnlySpan<byte> supports a builtin `+` operator.
+            return type.OriginalDefinition.Equals(this.CommonGetWellKnownType(WellKnownType.System_ReadOnlySpan_T)?.GetISymbol(), SymbolEqualityComparer.ConsiderEverything) &&
```

Comment:
>CommonGetWellKnownType [](http://example.com/codeflow?start=55&length=22)

Rather than going through the common layer, consider dropping to the C# specific layer instead, less translations will be involved.